edpm_ansible/roles/edpm_nftables/templates/chains.j2

```diff
@@ -0,0 +1,14 @@
+# Managed by edpm-ansible/edpm_nftables
+
+{% set chains = namespace(chains=[]) %}
+{%- for ruleset in edpm_nftables_rules %}
+{%    set rule=ruleset['rule'] %}
+{%    set chain_key = rule.get('chain', 'INPUT') ~ rule.get('table', 'filter') %}
+{%-   if chain_key not in chains.chains %}
+{%-     if 'extras' not in rule or rule['extras'].get('ensure', 'present') in ['enabled', 'present'] %}
+# Create chain EDPM_{{ rule.get('chain', 'INPUT') }} in {{rule.get('table', 'filter') }} table
+add chain inet {{ rule.get('table', 'filter') }} EDPM_{{ rule.get('chain', 'INPUT') }}
```

Comment:
as discussed, it may be better to get it as a parameter, imho

Reply:
ack